src/components/Operation.js

```diff
@@ -1,15 +1,41 @@
-const Operation = ({ value }) => {
+import styled from 'styled-components';
+
+const StyledOperation = styled.div`
+  & {
+    padding: 10px;
+    border-radius: 4px;
+    margin-bottom: 0.7rem;
+    cursor: pointer;
+    background-color: ${props => {
+      if (props.value === 'clear') return '#840032';
+      else if (props.value === '=') return '#002642';
+      else return '#E59500';
+    }};
+    background-image: -webkit-linear-gradient(283deg, rgba(255, 255, 255, 0.1) 50%, transparent 55%),-webkit-linear-gradient(top, rgba(255, 255, 255, 0.15), transparent);
+    box-shadow: ${props => {
```

Comment:
there are ways to compose styled components - seems like your buttons share a lot of common rules